src/Engine/Spells/Spells.cpp

```diff
@@ -1101,6 +1101,7 @@ void armageddonProgress() {
     pParty->uFlags |= PARTY_FLAGS_1_ForceRedraw;
     pParty->armageddon_timer -= pEventTimer->uTimeElapsed; // Was pMiscTimer
 
+    --pParty->armageddonForceCount;
```

Comment:
But doesn't `armageddonProgress` called every frame? This will not cause armageddonForceCount to quickly become 0?
Maybe we need not to do anything in this function if pEventTimer->uTimeElapsed is zero?

Reply:
I'll add a separate TODO for that.